server.py

```
@@ -519,6 +551,27 @@ def get_top_posts(
+                        post.comments.replace_more(limit=None)
+
+                        top_level_comments = [
+                            c
+                            for c in post.comments
+                            if isinstance(c, praw.models.Comment)
+                        ]
+
+                        post_data["comments"] = [
+                            _serialize_comment_tree(c) for c in top_level_comments
+                        ]
+                    except Exception as comments_error:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)